edgexfoundry-holding/device-onvif-camera
# refactor!: prefix some commands with onvif service name and add SnapshotUri

BREAKING CHANGE: Changed the name of many device commands to include a prefix of the onvif service they belong to. For example Status becomes PTZStatus, Profiles becomes MediaProfiles, etc.







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] **TODO** I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] **TODO** I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions


Make sure to **clean edgex**

- Build and run latest edgex main and this branch of device-onvif-camera
- Import the device-onvif-camera postman collection
- Import the postman env file if not done already
- Update postman env with your camera's values
- Run the postman collection
- Verify commands work as they did before

## New Dependency Instructions (If applicable)
